src/services/AuthService.ts

```diff
@@ -0,0 +1,25 @@
+import jwt, { JwtPayload } from "jsonwebtoken";
+import { secrets } from "../utils/secrets";
+
+const jwtPrivateKey = `-----BEGIN EC PRIVATE KEY-----\n${secrets.JWTPrivateKey}\n-----END EC PRIVATE KEY-----`;
+const jwtPublicKey = `-----BEGIN PUBLIC KEY-----\n${secrets.JWTPublicKey}\n-----END PUBLIC KEY-----`;
+const algorithm = "ES256";
+
+export class AuthService {
+  public decodeToken(token: string): JwtPayload {
```

Comment:
Just noticed that `decodeToken` could confuse people with our built-in `Token` stuff.
I'd rename it to `decodeJWT`